c_src/betree.c

```diff
@@ -859,6 +860,95 @@ static ERL_NIF_TERM nif_betree_search(ErlNifEnv* env, int argc, const ERL_NIF_TE
     return retval;
 }
 
```

Comment:
why you are not calling nif_betree_search inside nif_betree_search_t?

is checking input parameters and creating return tuple affect measurements a lot?

Reply:
I tried to bee as accurate as possible.  But I think your are right. Will change